test/e2e/specs/authorization-code-spec.js

```diff
@@ -41,7 +41,8 @@ describe('Authorization Code Flows', () => {
     expect(app.getEmail()).toBe('george@acme.com');
   });
 
-  it('can login with a custom login form', () => {
+  // TODO - Enable after hosted login-page flow is integrated 
+  xit('can login with a custom login form', () => {
```

Comment:
I wonder if we should just remove this? Seems like we will need to revisit this much later.

Reply:
The next step for this repo is to refactor it into the V2 strategy, where we will test this (though the code will probably differ a lot)

Reply:
Hosted login page for express is already in. So it won't be much long before we integrate it